packages/mattermost-redux/src/selectors/entities/cloud.ts

```diff
@@ -65,3 +65,10 @@ export function isCurrentLicenseCloud(state: GlobalState): boolean {
     const license = getLicense(state);
     return license?.Cloud === 'true';
 }
+
+export function isCloudTrialExpired(state: GlobalState): boolean {
```

Comment:
why don't use the existing one `checkHadPriorTrial`, basically does the same https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/cloud.ts#L59